lib/updatehub/updatehub.c

```diff
@@ -757,6 +757,12 @@ enum updatehub_response updatehub_update(void)
 		goto error;
```

Comment:
In commit `lib: updatehub: Add MCUboot request upgrade cmd`, the sentence "This add missing command." is not proper English and is difficult to understand, could you refrase the commit body a bit.

Reply:
Please, re-check and give me feedback.

Reply:
Looks good now